closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -75,7 +75,7 @@
     private byte[] keyBytes;
 
     // The nonce used for this object
-    private byte[] nonce;
+    private byte[] nonce = null;
```

Comment:
This can be removed please. The VM initialized fields to null by default, it doesn't need to be done explicitly which just adds byte codes.

Reply:
Sure, the commit is updated. Thanks.